packages/deploy-script-support/src/externalTypes.js

```diff
@@ -34,12 +28,12 @@ export {};
  */
 
 /**
- * @callback ProposalBuilder
+ * @callback CoreEvalBuilder
  * @param {{
  *   publishRef: PublishBundleRef,
  *   install: InstallEntrypoint,
  *   wrapInstall?: <T extends InstallEntrypoint>(f: T) => T }
  * } powers
  * @param {...any} args
- * @returns {Promise<ProposalResult>}
+ * @returns {Promise<{sourceSpec: string, getManifestCall: [exportedGetManifest: string, ...manifestArgs: any[]]}>}
```

Comment:
This is a **huge** improvement, but I still find myself frustrated by the names.
```suggestion
 * @returns {Promise<{sourceSpec: string, getManifestCall: [manifestGetterName: string, ...manifestGetterArgs: any[]]}>}
```
or possibly
```suggestion
 * @returns {Promise<CoreEvalDescriptor>}
```
with
```js
/**
 * @typedef CoreEvalDescriptor
 * @property {string} sourceSpec import specifier for a module
 * @property {[manifestGetterName: string, ...manifestGetterArgs: any[]]} getManifestCall
 *   the name of a function exported by the module and arguments to invoke it
 *   with in order to get a manifest (a Record that associates functions to be
 *   invoked and permits defining bootstrap-space powers they will have access
 *   to, see {@link ../README.md} and {@link runModuleBehaviors})
 */
```